### PR TITLE
Completed steps 1-4 of public feed familiarization project

### PR DESCRIPTION
**Pull request by ellikap6**
Added code to DataStrore and created MessageFeedServlet.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java
Comment (tsalako) [inline src/main/java/com/google/codeu/data/Datastore.java]: Nice helper function. Maybe instead of passing in the user name you can just pass in `datastore.prepare(query)`. so you dont have to do the if-statement before. Each method can handle its own query.

File: src/main/java/com/google/codeu/data/Datastore.java
Comment (tsalako) [inline src/main/java/com/google/codeu/data/Datastore.java]: i dont think you need this variable. the user specific query is already filtered by the query. so all the user messages have that username already `user = (String) entity.getProperty("user");` should be correct for this case as well.

**Review by tsalako — CHANGES_REQUESTED** [review]
Left two comments but this looks good!

**Review by tsalako — APPROVED** [review]

**Comment by tsalako** [comment]
Please wait for Raymond to approve before merging,

**Review by Rhaypapenfuzz — APPROVED** [review]

**Pull request merged**
